spec/21-http_response_header_format.md

```diff
@@ -52,18 +51,28 @@ trace-flags      = 2HEXDIGLC   ; 8 bit flags. Currently, only one bit is used. S
 
 #### trace-id
 
```

Comment:
the notion of trace-id in the response header should be identical to trace-in in `traceparent`. Why can we not link there instead of re-defining and potentially diverging?

Reply:
Good point, sure I have taken a pass at updating the description for a couple of fields (trace-id and one of the flags) to link to the corresponding fields in traceparent header. We likely want to keep the description for the sampled flag independently to express the callee/caller semantics easily.